kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -760,10 +795,12 @@ protected String appendExtractionExpression(String type, String name, String id,
             jsonTemplate.append("getDocumentData('")
                         .append(id)
                         .append("')");
-        } else if (type.equals("date")) {
-            jsonTemplate.append("getDateFormated('")
-                        .append(id)
-                        .append("')");
+        } else if (type.equals("Date")) {
```

Comment:
are you sure about this change of case?

Reply:
I just want to re-user https://github.com/kiegroup/droolsjbpm-integration/blob/8c89a1e3028721c02a97a71de77ecdd89653f6e3/kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java#L111